awscli/arguments.py

```diff
@@ -440,11 +438,11 @@ def add_to_params(self, parameters, value):
             parameters[self._serialized_name] = unpacked
 
     def _unpack_argument(self, value):
-        service_name = self.operation_object.service.endpoint_prefix
-        operation_name = xform_name(self.operation_object.name, '-')
+        service_name = self._operation_model.service_model.endpoint_prefix
```

Comment:
Actually, for this specific scenario, I'd be inclined to keep this as is for now.  The previous code used `endpoint_prefix` so to keep the changes to a minimum for the client migration, I'd be ok with keeping it the same for now.

I certainly think it would be worth standardizing on one vs. the other, but I don't think that necessarily has to happen with this PR.